components/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.1.9",
```

Comment:
Let's also increment the version number of the components in `components/package.json` and `components/package-lock.json` so they get pushed to npm